### PR TITLE
hw2

### PR DESCRIPTION
**Pull request by vitaliuchik**

### CODE REVIEW

**Comment by pavlo-berezin** [comment]
image
please fix console errors.

File: src/components/ContactPageContainer/ContactPageContainer.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
+input[type=text], input[type=email], select, textarea {
+    width: 100%;
+    padding: 12px;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+    box-sizing: border-box;
+    margin-top: 6px;
+    margin-bottom: 16px;
+    resize: none;
+  }
```
Comment (pavlo-berezin) [inline src/components/ContactPageContainer/ContactPageContainer.css:10]: please fix formatting.

File: src/components/ContactPageContainer/ContactPageContainer.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import './ContactPageContainer.css'
+import '../../App.css'
+
+
+
+function ContactPageContainer() {
```
Comment (pavlo-berezin) [inline src/components/ContactPageContainer/ContactPageContainer.js:7]: please fix formatting in this file.

File: src/components/Footer/Footer.css [hunk 3]
```diff
@@ -1,19 +1,15 @@
 .footer-container {
```
Comment (pavlo-berezin) [inline src/components/Footer/Footer.css:1]: There is Footer.css but no Footer.js? Why?

File: src/components/Header/Header.js [hunk 4]
```diff
@@ -4,16 +4,26 @@ import './Header.css'
...
+import {Link} from "react-router-dom";
+
 
 function Header() {
     return (
         <div className="header-container">
             <div id="title"><p>Tutora</p></div>
             <div className="header-button-container">
+                <Link to="/">
                 <div><Button variant="contained" className="button">Home</Button></div>
```
Comment (pavlo-berezin) [inline src/components/Header/Header.js:16]: Fix formatting.

File: src/components/Header/Header.js [hunk 5]
```diff
@@ -4,16 +4,26 @@ import './Header.css'
 import {Button} from '@material-ui/core'
 import LoginMenu from './LoginMenu'
 
+import {Link} from "react-router-dom";
```
Comment (pavlo-berezin) [inline src/components/Header/Header.js:7]: `... { Link } from ...`

File: src/components/JoinPageContainer/JoinPageContainer.js [hunk 6]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import './JoinPageContainer.css'
+import '../../App.css'
+
+import ModalContainer from '../Modal/ModalContainer'
+
+function JoinPageContainer() {
+    return (
+        <div className="join-container container">
+            <p>Info how to become a tutor</p>
```
Comment (pavlo-berezin) [inline src/components/JoinPageContainer/JoinPageContainer.js:10]: Maybe add some example info.

File: src/components/Modal/LoginModal.js [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+            <form action="" method="post">
+                <label for="uname" className="label-input"><b>Username</b></label>
+                <input type="text" placeholder="Enter Username" name="uname" required />
+
+                <label for="psw" className="label-input"><b>Password</b></label>
+                <input type="password" placeholder="Enter Password" name="psw" required />
+
+                <button type="submit">Login</button>
+                <label className="label-submit">
+                    <input type="checkbox" checked="checked" name="remember" /> Remember me
```
Comment (pavlo-berezin) [inline src/components/Modal/LoginModal.js:18]: You are putting inputs outside of labels. Why aren't you doing it here too?

File: src/components/Modal/Modal.css [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+    box-sizing: border-box;
+    border-radius: 20px;
+}
+
+.label-input {
+    margin-top: 50px;
+}
+
+.button {
+    background-color: #333333 !important;
```
Comment (pavlo-berezin) [inline src/components/Modal/Modal.css:22]: Don't use importants.

File: src/components/Modal/ModalContainer.js [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+    setOpen(true)
+  }
+ 
+  const onCloseModal = () => {
+    setOpen(false)
+  }
+
+  return (
+    <div>
+      {props.button === 'button' ? <Button onClick={onOpenModal} variant="contained" className="button">{props.label}</Button> : <MenuItem onClick={onOpenModal}>{props.label}</MenuItem>}
```
Comment (pavlo-berezin) [inline src/components/Modal/ModalContainer.js:24]: This line is too long. Break it to improve readability.

File: src/components/ContactPageContainer/ContactPageContainer.css (hunk 1 above)
Comment (vitaliuchik) [reply src/components/ContactPageContainer/ContactPageContainer.css:10]: explain

File: src/components/Header/Header.js (hunk 4 above)
Comment (vitaliuchik) [reply src/components/Header/Header.js:16]: explain

File: src/components/Header/Header.js (hunk 5 above)
Comment (vitaliuchik) [reply src/components/Header/Header.js:7]: what's wrong?

File: src/components/Modal/Modal.css (hunk 8 above)
Comment (vitaliuchik) [reply src/components/Modal/Modal.css:22]: easy to say, but Idk what to use instead. Here I have to change color of a button from material-ui

**Pull request merged**